docs/backup_and_restore.rst

```diff
@@ -472,7 +472,7 @@ source accounts, and journalist accounts. To do so, follow the steps below:
 
       cd ~/Persistent/securedrop/
       git fetch --tags
-      git tag -v 2.0.0
+      git tag -v 2.0.1
```

Comment:
Same as above - output should reflect new key (there may be an argument for moving this section into an include (and the version updater script would need to reflect that))